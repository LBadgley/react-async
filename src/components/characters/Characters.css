.Characters {
  display: flex;
  justify-content: space-around;
  flex-wrap: wrap;
  align-items: center;
  background-color: rgb(252, 252, 171);

  li {
    width: 20vw;
    margin: 2em;
    padding: 2rem;
    background-color: white;
    font-size: 16px;
    text-align: center;
    list-style-type: none;
  }

  section {
    margin: auto;
    background-color: white;

    color: black;
    animation: insert a linear;
  }

  img {
    width: 10rem;
    margin: auto;
  }

  @keyframes insert {
    from {
      width: 0;
    }
    to {
      width: 100vw;
    }
  }
}
