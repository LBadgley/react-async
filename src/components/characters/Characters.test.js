import React from 'react';
import { shallow } from 'enzyme';
import Characters from './Characters';

describe('characters component', () => {
  it('renders the characters component', () => {
    const characters = [
      {
        name: 'Morty Smith',
        status: 'Alive',
        species: 'Human',
        image: './image'
      },
      {
        name: 'Summer Smith',
        status: 'Alive',
        species: 'Human',
        image: './image'
      }
    ];
    const wrapper = shallow(<Characters characters={characters}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
