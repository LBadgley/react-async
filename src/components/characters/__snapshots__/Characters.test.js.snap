// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`characters component renders the characters component 1`] = `
<ul
  className="Characters"
>
  <li
    key="0"
  >
    <Character
      character={
        Object {
          "image": "./image",
          "name": "Morty Smith",
          "species": "Human",
          "status": "Alive",
        }
      }
    />
  </li>
  <li
    key="1"
  >
    <Character
      character={
        Object {
          "image": "./image",
          "name": "Summer Smith",
          "species": "Human",
          "status": "Alive",
        }
      }
    />
  </li>
</ul>
`;
